news/2018-11-04-project-loved-week-of-november-4th.md

```diff
@@ -0,0 +1,215 @@
+
+Some people may recognize this mapper from another Loved map, "Imaginary Places", which is iconic for its boss level difficulty. This map is no different. Clocking in over 8 stars, this alt-styled fast aim map by LukerMaster will push you to your limits. Good luck!
+
+[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/black-or-white.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[BlackYooh vs. siromaru - BLACK or WHITE?](https://osu.ppy.sh/beatmapsets/463454#osu) by [Akali](https://osu.ppy.sh/users/2127359)  
+*written by [Bubbleman](https://osu.ppy.sh/users/5182050)*
+
+If you've been in the tournament scene long enough, you'll have seen this played as a tiebreaker more than a few times, and for good reason. It balances flow and snap aim nicely, includes some cut streams to keep you on your toes, and weighing in at over five minutes, it is the perfect length to test your focus. It's all topped off with a banger of a song, so be sure to give it a look-see!
+
+[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/f-zero-big-blue.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[S.S.H. - F-zero ~ Big Blue](https://osu.ppy.sh/beatmapsets/154798#osu) by [Saten](https://osu.ppy.sh/users/444506)  
+*written by [Bubbleman](https://osu.ppy.sh/users/5182050)*
+
+An iconic song meeting an iconic mapping style, Saten's map of S.S.H.'s Big Blue combines older style patterning with an artist we don't see much of on osu! these days. As such it has a nostalgic feel, but don't let that distract you from the 228 BPM streams and frequent switches between 1/4 and 1/3 rhythms. A lower difficulty has been recently added too, so if you're after a more modern feel with the same Saten style, this is a mapset you don't want to miss—a sentiment shared by at least 60 others.
+
+[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/stay-for-a-while.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[Virtual Riot - Stay For A While](https://osu.ppy.sh/beatmapsets/690556#osu) by [ProfessionalBox](https://osu.ppy.sh/users/3250792) and [cRyo[iceeicee]](https://osu.ppy.sh/users/835998)  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+Yet another ProfessionalBox map has made its way to the Loved voting stage! This time, it's a control-crazy map of Virtual Riot's "Stay For A While" designed to throw you off with its confusing slider shapes and insanely high slider tick rate of 4. Tied along with quite a catchy and energetic future bass song, this map does an amazing job creating a unique playing experience—I'm sure the 120+ favorites can agree with me on that! So, what say you? Think you've got the cursor control it takes to pass a map like this? If you said yes, I'm going to need you to stop reading this and give the map your best go. Trust me, it'll be worth it! Afterwards, come back to this page and drop a vote on the map's poll!
```

Comment:
```suggestion
Yet another ProfessionalBox's beatmap has made its way to the Loved voting stage! This time, it's a control-crazy map of Virtual Riot's "Stay For A While" designed to throw you off with its confusing slider shapes and insanely high slider tick rate of 4. Tied along with quite a catchy and energetic future bass song, this beatmap does an amazing job creating a unique playing experience—I'm sure the 120+ favorites can agree with me on that! So, what say you? Think you've got the cursor control it takes to pass a map like this? If you said yes, I'm going to tell you to stop reading this and give the map your best go. Trust me, it'll be worth it! Afterwards, come back to this page and drop a vote on the beatmap's poll!
```

Reply:
#osu-wiki and I think "need" is just part of digitalhypno's "go play this!!" style so no change